src/web/app/components/question-text-with-info/__snapshots__/question-text-with-info.component.spec.ts.snap

```diff
@@ -80,13 +81,13 @@ exports[`QuestionTextWithInfoComponent should show question detail when click co
   <span
     class="d-inline-flex align-items-center"
   >
-    <span
-      class="col-auto"
+    <h3
+      class="col-auto question-number"
     >
       <b>
         Question 0: 
       </b>
```

Comment:
Use of <b> tags has been discouraged for some time already. <b> tag in HTML 5 has been redefined (for legacy reasons) but does not stand for bold anymore. HTML is for elements that convey structural meaning, presentational elements (such as font weight and font size) should be in css now.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

Reply:
Interesting, did not know about this! Will change to use CSS instead. Thanks!